cluster/gce/trusty/node.yaml

```diff
@@ -262,6 +262,35 @@ end script
 MIME-Version: 1.0
 Content-Type: text/upstart-job; charset="us-ascii"
 Content-Transfer-Encoding: 7bit
+Content-Disposition: attachment; filename="kube-install-additional-packages.conf"
+
+#upstart-job
+
+description "Install additional packages used by kubernetes"
+
+start on stopped kube-docker
+
+script
+	# Socat and nsenter are used in port forwarding. They are not required for spin
+	# up a cluster. Therefore, we move the installation here to make it parallel to
+	# other steps to reduce the cluster creation time.
```

Comment:
The difference is that we have a way to pretty easily check if all of the kube-system containers are up (/componentstatuses) which is part of validate-cluster.sh. So unless that passes, the cluster won't be marked healthy.